core/sql/executor/ExExeUtilLoad.cpp

```diff
@@ -1442,7 +1442,8 @@ short ExExeUtilHBaseBulkLoadTcb::work()
            ComCondition *cond;
            Lng32 entryNumber;
            while ((cond = diagsArea->findCondition(EXE_ERROR_ROWS_FOUND, &entryNumber)) != NULL) {
-              errorRowCount = cond->getOptionalInteger(0);
+              if (errorRowCount < cond->getOptionalInteger(0))
```

Comment:
Actually, the changes in ExHdfsScan.cpp is the one needed to take care of the core dump issue.  To honor the command 'stop after n error rows' when the data is loaded into trafodion table, a counter is maintained in hbase to increment the number of error rows from all the ESP process.  Each load prepare command has its own counter id. This counter id is visible to the ExHbaseAccessBulkLoadPrepSQTcb operator. This operator hosted in multiple ESPs will read the counter at the end of its operation and pass it as a warning. So all the ESPs are expected to have the same value but if an esp finishes early it may not have read the final counter value. Hence I am taking the max error counter value passed in the warnings.